frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -209,7 +254,59 @@ pub mod pallet {
 
 	impl<T: Config> Pallet<T> {
 		#[transactional]
-		fn do_rebalance(_vault_id: &T::VaultId) -> DispatchResult {
+		fn do_rebalance(vault_id: &T::VaultId) -> DispatchResult {
+			let asset_id = T::Vault::asset_id(vault_id)?;
+			let account_id = T::Vault::account_id(vault_id);
+			let pool_id = Self::pools(asset_id).ok_or_else(|| Error::<T>::PoolNotFound)?;
+			let task = T::Vault::available_funds(vault_id, &Self::account_id())?;
+			match task {
+				FundsAvailability::Withdrawable(balance) => {
+					let vault_account = T::Vault::account_id(vault_id);
+					let lp_token_amount = T::Pablo::amount_of_lp_token_for_added_liquidity(
+						pool_id,
+						T::Balance::zero(),
+						balance,
+					)?;
+					T::Pablo::add_liquidity(
+						&vault_account,
+						pool_id,
+						T::Balance::zero(),
+						balance,
+						lp_token_amount,
+						bool::default(),
+					)?;
+				},
+				FundsAvailability::Depositable(balance) => {
+					let vault_account = T::Vault::account_id(vault_id);
+					let lp_token_amount = T::Pablo::amount_of_lp_token_for_added_liquidity(
+						pool_id,
+						T::Balance::zero(),
+						balance,
+					)?;
+					T::Pablo::remove_liquidity(
+						&vault_account,
+						pool_id,
+						lp_token_amount,
+						T::Balance::zero(),
+						T::Balance::zero(),
+					)?;
+				},
+				FundsAvailability::MustLiquidate => {
+					let vault_account = T::Vault::account_id(vault_id);
+					let lp_token_id = T::Pablo::lp_token(pool_id)?;
+					let balance_of_lp_token = T::Currency::balance(lp_token_id, &account_id);
+					T::Pablo::remove_liquidity(
+						&vault_account,
+						pool_id,
+						balance_of_lp_token,
+						T::Balance::zero(),
+						T::Balance::zero(),
+					)?;
+				},
+				FundsAvailability::Equilibrable => {
+					();
+				},
```

Comment:
```suggestion
				FundsAvailability::Equilibrable => {},
```

You don't have to use unit (`()`) here.